### PR TITLE
Resolve ts-loader

### PR DESCRIPTION
**Pull request by jameslnewell**
so its found in a yarn workspaces monorepo

## What was changed
Wrapped `ts-loader` in a call to `require.resolve()`

## Why?
So its correctly located in a `yarn workspaces` monorepo

## Checklist

1. Closes 

2. How was this tested:
I edited the published source code to make this change and re-ran the sample app I was working in.

3. Any docs updates needed?
N/A

### CODE REVIEW

**Comment by lorensr** [comment]
Thanks for the contribution! It appears as though the recommended way to get [`loader`](https://webpack.js.org/configuration/module/#useentry) to resolve correctly is by setting [`context`](https://webpack.js.org/configuration/entry-context/#context), like this to get from `worker/src/workflow/` to `worker/`:

```ts
context: path.resolve(__dirname, '../..')
```

Would that solve your issue?

**Comment by jameslnewell** [comment]
Yeah that works too

**Review by lorensr — APPROVED** [review]

File: packages/worker/src/workflow/bundler.ts [hunk 1]
```diff
@@ -117,6 +117,7 @@ export class WorkflowCodeBundler {
           },
         ],
       },
+      context: path.resolve(__dirname, '../..'),
```
Comment (bergundy) [inline packages/worker/src/workflow/bundler.ts:120]: This should usually be the directory where the app is located, not our SDK.
Can we `require.resolve` the `ts-loader` instead?

Comment (bergundy) [reply packages/worker/src/workflow/bundler.ts:120]: @lorensr WDYT?

Comment (jameslnewell) [reply packages/worker/src/workflow/bundler.ts:120]: @bergundy that's [what I had originally](https://github.com/temporalio/sdk-typescript/pull/492/commits/22ebb5c03a281c62b636cc34e02693a1906512d5) 😆

Comment (lorensr) [reply packages/worker/src/workflow/bundler.ts:120]: Ah. Yeah, I guess that's better than having user provide their root dir to put for context. Sorry James 😄

**Comment by bergundy** [comment]
Thanks @jameslnewell!

**Pull request merged**